templates/parts/credit.php

```diff
@@ -26,7 +35,11 @@
  */
 if ( (bool) apply_filters( 'primer_author_credit', true ) ) {
 
-	echo ' &mdash; ';
+	if ( ! empty( $copyright_text ) ) {
```

Comment:
Done in https://github.com/godaddy/wp-primer-theme/pull/157/commits/f25705dd445afe9a8264bfd418285ef3da6bb508